core/settings.yaml

```diff
@@ -5,7 +5,8 @@ environment:
   action_space_range: 2  # Number of possible actions the agent can take in the environment
   number_of_agents: 25 # Number af agents in environment starting
```

Comment:
In the future, I don't think we should have this defined in two separate places, here and in godot. But that is something for later.